row_002_app.py

```
@@ -0,0 +1,451 @@
+        int(request.form.get('identifier')),
+        int(request.form.get('font')),
+        int(request.form.get('text_width')),
+        int(request.form.get('scroll_speed')),
+        int(request.form.get('scroll_direction'))
+    )
+
+    return 'OK'
+
+
+def _reset_gif():
+    return requests.post(f'http://{pixoo.ip_address}/post', json.dumps({
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)